### DISCUSSION TITLE
About increasing the spot request

### DISCUSSION THREAD

**Discussion by akr9757** [learndevopsonline/learndevopsonline#1676] [Q&A]
I am unable to increase spot request
![Screenshot_2023-12-15-17-32-11-970_com google android gm](https://github.com/learndevopsonline/learndevopsonline/assets/131577072/e2cf4118-a456-478e-91c3-adb2bf3d721b)

and also Even if I try to launch normal instances also it is failing
![Uploading 17026421063483032888458331604882.jpg…]()

**Comment by sreedharm07** [comment]
go through this doc

https://learndevopsonline.github.io/learndevopsonline/build/docs/How-Tos/increase-spot-quota

**Reply by akr9757** [reply]
I done the same bro
![17026566549141257821708428396356](https://github.com/learndevopsonline/learndevopsonline/assets/131577072/bdd59c99-ab44-453c-b490-c2d8ed6aef1b)

**Reply by sreedharm07** [reply]
ok .
 i dont have idea for next steps after  aws team only replied as cont increase.
join lab session and ask raghu may be he know next steps to do.

**Comment by akr9757** [comment]
Got the reply from raghu
